SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -309,6 +324,19 @@ TSharedRef<SWidget> FSpatialGDKEditorToolbarModule::CreateGenerateSchemaMenuCont
 	return MenuBuilder.MakeWidget();
 }
 
+TSharedRef<SWidget> FSpatialGDKEditorToolbarModule::CreateLaunchDeploymentMenuContent()
+{
+	// First parameter is whether to close the menu's window after a selection has been made.
+	FMenuBuilder MenuBuilder(true, PluginCommands);
```

Comment:
Ah I meant like this.
```suggestion
	FMenuBuilder MenuBuilder(true /*bInShouldCloseWindowAfterMenuSelection*/, PluginCommands);
```